src/env.js

```diff
@@ -175,6 +179,10 @@ export class Env {
     get exists() {
 
         const existsProxy = new Proxy(this.source, {
+            /**
+             * @param {any} target 
+             * @param {string} key 
```

Comment:
If we want to be 100% correct, key can also be a symbol.

Reply:
Yes, `Proxy` would accept a `symbol` key, but we can't, since `this.source` only has key strings in practice.  I mean... maybe you _could_ stuff a symbol in `process.env` but I'm not sure how you'd do that from bash 😄

Reply:
Keep in mind that source can be passed directly into the Env constructor. It doesn’t have to be read from the environment.